src/CloudFlareTimestampInterface.php

```diff
@@ -0,0 +1,21 @@
+<?php
+/**
+ * @file
+ * Contains \Drupal\cloudflare\CloudFlareTimestampInterface.
+ */
+
+namespace Drupal\cloudflare;
+
+/**
+ * Provides an injectable facility for getting the current time.
+ */
+interface CloudFlareTimestampInterface {
+  /**
+   * Gets the current DateTime.
+   *
+   * @return DateTime
+   *   DateTime representing the current time.
```

Comment:
That's PHP's ``DateTime`` right, in that case write ``\DateTime``, like in all other mentions of PHP references: ``\Exception``, ``\Countable``, ``\LogicException``, ``\Iterator``... etc